crates/store/re_chunk_store/src/query.rs

```diff
@@ -587,27 +590,106 @@ impl ChunkStore {
         chunks
     }
 
-    /// Returns the most-relevant _temporal_ chunk(s) for the given [`LatestAtQuery`].
+    /// Returns the most-relevant chunk(s) for the given [`LatestAtQuery`].
     ///
-    /// The returned vector is guaranteed free of duplicates, by definition.
+    /// Optionally include static data.
     ///
     /// The [`ChunkStore`] always work at the [`Chunk`] level (as opposed to the row level): it is
     /// oblivious to the data therein.
     /// For that reason, and because [`Chunk`]s are allowed to temporally overlap, it is possible
     /// that a query has more than one relevant chunk.
     ///
+    /// This is a multi-component query, so it may yield duplicate chunks, but any given chunk
```

Comment:
> This is a multi-component query, so it may yield duplicate chunks

Shouldn't it dedupe them before returning then? I'm not sure what this endpoint is for, but I suspect that every downstream consumer of this will want to dedupe the data, if only for performance reason (e.g. don't try and fetch the same chunk twice from a remote redap store)?

> but any given chunk will never contain both _static_ and _temporal_ data.

I would remove this, as this is nothing specific to this function: that's just how chunks work (and as someone who knows how chunks work, this confuses me into thinking something weird is happening, when it in fact isn't).

Reply:
> I'm not sure what this endpoint is for, but I suspect that every downstream consumer of this will want to dedupe the data, if only for performance reason

Right now, it's only for the time range query in redap. I agree that de-duplicating probably makes sense.